### DISCUSSION TITLE
New User With Questions

### DISCUSSION THREAD

**Discussion by ThinAirPhoto** [aaronwmorris/indi-allsky#1540] [Q&A]
Hi,

I am very excited about this project, and I have almost everything set to deploy my camera and start testing it out under the stars. I have a few issues and questions though that I cannot seem to figure out yet on my own, so hopefully quick to fix...

First, some background on my hardware/software setup:
- Raspberry Pi 4, 4GB. Already was being used for another purpose, but repurposed for this. Has latest 64 bit Bookworm OS installed
- Build all went OK
- Using Raspberry PiHQ camera
- 3D printed enclosure, will try with onboard Pi WiFi first, and then will change over to a Ethernet PoE feed if required instead. So far indoors WiFi is OK

Problems/Questions

1) I stupidly did not write down the username/password that was prompted at some point during setup (DOH!) I am confident I recall the password but not the username apparently. Is there a way to reset this? This is the for the credentials on the Indi-allsky webpage. If not, how far back in the build and install process would I have to go to wipe this and start over?

2) I can launch the indi-allsky service manually when directly on the RPi using 'systemctl --user start indi-allsky' from the indi-allsky folder, but not when logged in remotely via an ssh session to the RPi. This would seem like a requirement to get working once the RPi is packed up and up on the roof...

3) I am able to connect to the webpage using my windows desktop machine and the <indi-allsky raspberry Pi IP address>:443, but it is telling me it is not a secure connection. Assuming I am missing some step here either in setting things up initially or in the connection process?

4) I seem to recall reading about issues with the green tint to the output from the raspberry PI HQ camera but do not recall seeing a solution? Looks great when run through SCNR in PixInsight, but assuming needs to be fixed on the Pi to allow for good timelapses, etc...

Sorry for the newbie questions, hopefully a some easy answers...

ML

**Comment by aaronwmorris** [comment]
Fantastic!  I will help however I can.

> I stupidly did not write down the username/password that was prompted at some point during setup (DOH!) I am confident I recall the password but not the username apparently. Is there a way to reset this? This is the for the credentials on the Indi-allsky webpage. If not, how far back in the build and install process would I have to go to wipe this and start over?

You can always reset the password or create a new user.  This is the wiki page for password management:
https://github.com/aaronwmorris/indi-allsky/wiki/Web-Interface-Password

> I can launch the indi-allsky service manually when directly on the RPi using 'systemctl --user start indi-allsky' from the indi-allsky folder, but not when logged in remotely via an ssh session to the RPi. This would seem like a requirement to get working once the RPi is packed up and up on the roof...

I have only seen this problem happen when using a terminal emulator like screen, tmux, or byobu.  In any case, you can control the service via the web interface under System -> Services

> I am able to connect to the webpage using my windows desktop machine and the :443, but it is telling me it is not a secure connection. Assuming I am missing some step here either in setting things up initially or in the connection process?

This is just due to using a self-signed certificate.  I realize this is an irritation, but it is rather difficult to work around this.  It is not a real indication of a problem, it is really just a warning.

> I seem to recall reading about issues with the green tint to the output from the raspberry PI HQ camera but do not recall seeing a solution? Looks great when run through SCNR in PixInsight, but assuming needs to be fixed on the Pi to allow for good timelapses, etc...

indi-allsky actually supports 2 of the SCNR algorithms from PixInsight.  Under the Image tab in the config, about half way down, you can select either `Average Neutral` or `Maximum Neutral`.

**Reply by ThinAirPhoto** [reply]
Aaron - Excellent, thanks so much for the prompt reply with great suggestions, and for all your great work on this. I'm a retired EE/Software Engineer, so I can appreciate all the work that went into this amazing piece of code! I am sure I will have to get some sort of compensation your way once this is all running :)

**Reply by aaronwmorris** [reply]
Just don't look at the code too closely.  You might find out it is worth exactly how much you paid for it.  :-)
